src/components/StoreDeployBlock.js

```diff
@@ -3,33 +3,48 @@ import PropTypes from "prop-types";
 import HashColor from "./UI/HashColor";
 import Input from "./UI/Input";
 import { OrangeButton } from "./UI/Button";
+import { isEmpty } from "../components/utils";
 
 class StoreDeployBlock extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      issuerName: ""
+      issuerName: "",
+      issuerNameIsValid: true
     };
-
     this.onNameChange = this.onNameChange.bind(this);
     this.onDeployClick = this.onDeployClick.bind(this);
   }
 
   onNameChange(event) {
     this.setState({
-      issuerName: event.target.value
+      issuerName: event.target.value,
+      issuerIsValid: isEmpty(event.target.value)
     });
   }
 
   onDeployClick() {
     const { adminAddress, handleStoreDeploy } = this.props;
-    handleStoreDeploy({
-      fromAddress: adminAddress,
-      name: this.state.issuerName
-    });
+    const { issuerName } = this.state;
+    if (!isEmpty(issuerName)) {
+      console.log(adminAddress);
```

Comment:
console.log ?

Reply:
let's add an eslint rule to avoid that ?

Reply:
still there, no eslint rule

Reply:
For console,
"off" or 0 - turn the rule off
"warn" or 1 - turn the rule on as a warning (doesn’t affect exit code)
"error" or 2 - turn the rule on as an error (exit code will be 1)

But we are currently using console.error for failure in try and catch, so console cannot be disabled. There doesn't seem to be any rule to only allow console.error.

Reply:
can use logger instead
https://github.com/OpenCerts/admin-website/blob/master/src/logger.js